proxy/proxy.go

```diff
@@ -37,6 +37,8 @@ type Options struct {
 	AuthenticateInternalAddr string   `envconfig:"AUTHENTICATE_INTERNAL_URL"`
 	OverideCertificateName   string   `envconfig:"OVERIDE_CERTIFICATE_NAME"`
 	AuthenticatePort         int      `envconfig:"AUTHENTICATE_SERVICE_PORT"`
+	CA                       string   `envconfig:"CERTIFICATE_AUTHORITY"`
```

Comment:
I think `CERTIFICATE_AUTHORITY` and `CERTIFICATE_AUTHORITY_FILE` are fine names for the setting.